src/hooks/useNearFoodParty.ts

```diff
@@ -42,17 +42,28 @@ const createFoodPartyOverlay = ({ placeName, storeId }: NearFoodPartyItem) => {
 };
 
 const useNearFoodParty = () => {
-  const { kakaoMap, setKakaoMap } = useKakaoMapContext();
+  const { kakaoMap } = useKakaoMapContext();
   const [nearFoodParty, setNearFoodParty] = useState<NearFoodPartyItem[]>([]);
-  const [foodPartyOverlay, setFoodPartyOverlay] = useState<kakao.maps.CustomOverlay[]>(
-    []
-  );
   const [clickedRestaurant, setClickedRestaurant] = useState<Restaurant>();
 
   const getNearFoodParty = async (props: NearFoodPartyProps) => {
```

Comment:
ask: props라는 이름으로 받는 이유가 궁금해요~!

Reply:
타입이 NearFoodPartyProps라서 props로 지었던 거 같아요! props는 컴포넌트에서만 쓰는 거니까 혼란스러울 수 있으니까 물어보신거 맞나용? 흠...추천해주실만한 네이밍 있을까요? 위,경도, distance가 담긴 타입입니다!

Reply:
정확히 무엇 때문에 혼란스러운지 몰랐는데 말씀하신 것처럼 컴포넌트에서만 props라는 이름으로 쓰이다보니 혼란스러웠던 것 같아요.(다른 데에서도 props라고 사용하는지는 사실 잘 모르겠습니다)

```suggestion
  const getNearFoodParty = async ({ latitude, longitude, distance }) => {
```

마땅한 이름이 없다면 저렇게 객체를 통해 받는 것도 괜찮을 것 같습니다. 물론 return 문에서 ...props와 같이 간편히 처리할 수는 없을 것 같네요!

제가 지금 자세한 문맥을 몰라서 네이밍을 보고 혼란스러워 하는 것일수도 있습니다. 마땅한 이름이 없다면 그대로 반영해도 될 만큼 사소한 문제라고 봅니다.

Reply:
> 물론 return 문에서 ...props와 같이 간편히 처리할 수는 없을 것 같네요!

`...props` 이거때문에 한번에 받고싶었는데, 일단 props 외에 다른 네이밍을 한번 고민해보고 커밋해두겠습니다!

Reply:
params로 변경했고, props 쓰는 다른 곳도 수정했습니다! [24673f3](https://github.com/prgrms-web-devcourse/Team-DarkNight-Kkini-FE/pull/179/commits/24673f31692fa68b9ef4cf3dfd3d5bc86f22bb2b)